apps/src/code-studio/pd/workshop_survey/Demographics.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {FormGroup} from 'react-bootstrap';
+import FormComponent from '../form_components/FormComponent';
+
+const LABELS = {
+  "race": "Race (check all that apply)",
+  "yearsTaught": "If you are/were a K-12 teacher, how many years have you taught?",
+  "highestEducation": "What is your highest level of education?",
+  "degreeField": "What field of study is/are your degree(s) in?",
+  "yearsTaughtStem": "For how many years have you taught a STEM subject?",
+  "yearsTaughtCs": "For how many years have you taught Computer Science?",
+  "haveRequiredLicenses": "Do you have the required licenses / certifications to teach CS in your school district?"
+};
```

Comment:
Per offline discussion, these are passed in as data attributes from the server [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/local_summer_workshop_survey.rb#L50) (so they can be validated). 👍